app/AndroidManifest.xml

```diff
@@ -16,7 +16,14 @@
         package="net.openid.appauthdemo" >
 
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES" />
+
+    <queries>
```

Comment:
I am not sure if this is a good idea because https://github.com/openid/AppAuth-Android/pull/469

Reply:
is there any documentation on Opera, would be nice with a list of supported categories by Opera.
I would actually think that it should instead be a bug report to Opera, as it should support android.intent.category.BROWSABLE.
It has even been around for a long time... https://developer.android.com/reference/android/content/Intent#CATEGORY_BROWSABLE

Reply:
I don't know. I don't use Opera.

Reply:
Seems Opera is supporting https://developer.android.com/reference/android/content/Intent#CATEGORY_APP_BROWSER
Will allow both and add a check for it in the BrowserSelector.

Reply:
Not all browsers seem to support it, so I am still letting it also check for https://developer.android.com/reference/android/content/Intent#CATEGORY_BROWSABLE